pkg/util/util.go

```diff
@@ -640,3 +642,20 @@ func HttpGetFreePort() (int, error) {
 	}
 	return freePort, nil
 }
+
+// HTTPGetRequest uses url to get file contents
+func HTTPGetRequest(url string) ([]byte, error) {
+	var httpClient = &http.Client{Timeout: 10 * time.Second}
```

Comment:
Maybe we should make this timer a constant + throw it up at the top of the file?

Reply:
Updated, all tests will be added on my followup PR which is mainly for `odo create component <component>` since some codes may change on my followup PR affect the tests. But will do that for sure and let you review again.